### PR TITLE
fix asset stream

### PR DESCRIPTION
**Pull request by Tormund**

### CODE REVIEW

File: android/ndk/aasset_manager.nim [hunk 1]
```diff
@@ -65,12 +65,19 @@ proc aGetPosition(s: Stream): int =
...
+    let size = csize(slice.b + 1 - slice.a)
+    result = s.asset.read(addr buffer[slice.a], size).int
+
 proc aReadData(s: Stream, buffer: pointer, bufLen: int): int =
     let s = AssetStream(s)
     result = s.asset.read(buffer, csize(bufLen)).int
 
 proc aWriteData(s: Stream, buffer: pointer, bufLen: int) =
-    raise newException(Defect, "Can not write to asset")
+    var e = new(Defect)
```
Comment (yglukhov) [inline android/ndk/aasset_manager.nim]: Why this change?

Comment (Tormund) [reply android/ndk/aasset_manager.nim]: cleanuped
